controllers/nodemailer.js

```diff
@@ -0,0 +1,15 @@
+const sendMail = require("../utils/Nodemailer");
+
+const contactUs = (req, res) => {
+
+  const data = {
+    email: req.body.email,
+    message: req.body.message,
+    name: req.body.name
+  }
+
+  sendMail(data)
+  res.send("succes")
```

Comment:
This is not the most accurate way to write this controller, what if the user does not provide an email, or a name,(I know you made it so that the user is required to write these fields in the frontend but think about insomnia/postman, they are not confined to use the frontend to send a request to the api) the sendMail would fail, which would crash the server, there's a file called ApiError in the utils folder, this a custom error you could throw by doing
throw new ApiError(STATUSCODE,MESSAGE)
please check for missing information and throw the relevant error.

Also, success is always sent in this request, but what if the sendMail failed to send the error, we would never know, please make it so that the sendMail returns a boolean, true for if the email was sent, false if along the way something went wrong, and send the appropriate response back in the res.send()